packages/mui-base/src/Dialog/Title/DialogTitle.tsx

```diff
@@ -0,0 +1,60 @@
+import * as React from 'react';
+import PropTypes from 'prop-types';
+import type { DialogTitleProps } from './DialogTitle.types';
+import { useComponentRenderer } from '../../utils/useComponentRenderer';
+import { useDialogRootContext } from '../Root/DialogRootContext';
+
+const DialogTitle = React.forwardRef(function DialogTitle(
+  props: DialogTitleProps,
+  forwardedRef: React.ForwardedRef<HTMLParagraphElement>,
+) {
+  const { render, className, id, ...other } = props;
+  const { setTitleElementId, open, modal } = useDialogRootContext();
+
+  const ownerState = {
+    open,
+    modal,
+  };
```

Comment:
`memo`?

Reply:
Technically it doesn't matter, as the whole ownerState isn't a dependency (just its fields)